src/test/java/com/stripe/BaseStripeFunctionalTest.java

```diff
@@ -72,8 +72,13 @@ public static Customer createDefaultCustomerWithPlan(Plan plan)
     @BeforeClass
     public static void setUp() {
         Stripe.apiKey = "tGN0bIwXnHdwOa85VABjPdSn8nWY7G7I"; // stripe public
+
+        // Peg the API version so that it can be varied independently of the
+        // one set on the test account.
+        Stripe.apiVersion = "2017-02-14";
+
         // test key
-        supportedRequestOptions = RequestOptions.builder().setStripeVersion("2015-02-16").build();
+        supportedRequestOptions = RequestOptions.builder().setStripeVersion(Stripe.apiVersion).build();
```

Comment:
I'm honestly not 100% sure what the intention of this variable was. I _think_ that it was maybe to assemble some basic options and test it against the occasional request. It seems a little superfluous though TBH, and we may want to strip it out on another PR at some point.